app/components/special-coverage/special-coverage-edit/special-coverage-edit-directive.js

```diff
@@ -14,7 +14,7 @@ angular.module('specialCoverage.edit.directive', [
 ])
   .directive('specialCoverageEdit', function (routes) {
     return {
-      controller: function (_, $location, $q, $scope, Campaign, EXTERNAL_URL,
+      controller: function (_, $location, $q, $scope, $http, Campaign, EXTERNAL_URL,
```

Comment:
We should prune any of the injection dependencies we're not using here like `Campaign`